controllers/storagecluster/provider_server.go

```diff
@@ -42,6 +42,7 @@ const (
 
 	ocsClientConfigMapName = "ocs-client-operator-config"
 	deployCSIKey           = "DEPLOY_CSI"
+	manageNoobaaSub        = "MANAGE_NOOBAA_SUB"
```

Comment:
Lets not use short names in keys 
```suggestion
	manageNoobaaSub        = "MANAGE_NOOBAA_SUBSCRIPTION"
```